Cython/Compiler/Tests/TestTreeFragment.py

```diff
@@ -59,6 +60,26 @@ def test_temps(self):
         self.assertTrue(isinstance(s[1].rhs, TempRefNode))
         self.assertTrue(s[0].expr.handle is s[1].rhs.handle)
 
+    def test_parse_fault_tolerant_complete(self):
+        code = u'''
+def method():
+    pass
+
+def method2():
+    error_here =
+
+def method3():
+    error_here =
+
+def method4():
+    without_error_here = 10
+'''
+        with local_errors() as errors:
+            parse_from_strings("test_name", code, True)
+
+        self.assertEqual("CompileError((<StringSourceDescriptor:test_name>, 6, 16), 'Expected an identifier or literal')", repr(errors[0]))
```

Comment:
Please also check that `len(errors) == 3` and that `error[1]` is also what you expect.

Reply:
This was missed in the updated.

Reply:
(the `len()` part)

Reply:
Alright